core/spec.md

```diff
@@ -1086,37 +1088,60 @@ Content-Type: application/json; charset=utf-8
 
 ### Registry Model
 
-The Registry model defines extension attributes, Groups and Resources supported
-by the implementation of the Registry Service. This information will usually
-be used by tooling that does not have advance knowledge of the type of data
-stored within the Registry.
-
-When a new implementation is first created, or a new specification is based
-on the xRegistry specification, then it MAY choose to modify the core
-attributes to fit its needs. The changes allowed are restricted to the
-following:
-- OPTIONAL attributes MAY be defined as REQUIRED
-- The `description` of an attribute MAY be defined
-- `ifvalues` specifications MAY be defined
-
-To indicate a change to a core attribute, the attribute MUST be defined
-as part of the Registry model. Once a Registry instance is created, further
-changes to these core attribute MUST adhere to the rules defined in the
-[Updating the Registry Model](#updating-the-registry-model) section.
-
-Any core attributes that are not defined in the model of an instance of a
-Registry MUST be implicitly defined. In other words, the Registry's model
-consists of the core attributes (as defined by this specification) overlaid
-with the core and extension attributes that are explicitly defined as part of
-its model.
+The Registry model defines the Groups, Resources, extension attributes and
+changes to specification defined core attributes. This information is
+intended to be used by tooling that does not have advance knowledge of the
```

Comment:
I initially thought "advance" was a typo for "advanced", but I now understand it to be more "in advance". Perhaps "does not have knowledge of the structure of the Registry in advance" to make it easier to parse?